### ISSUE TITLE
[version: 0.4.0] Problems with the path to the files

### ISSUE THREAD

**Issue by TheOdemMove** [7nohe/openapi-react-query-codegen#21]
After generate client with openapi-react-query-codegen i get not quite the expected result.  

The windows platform uses `\\` in paths, for example:   
```Typescript
import { useQuery, useMutation, UseQueryOptions, UseMutationOptions } from "@tanstack\\react-query";
import { ValidationError } from "..\\requests\\models\\ValidationError";
import { ResponseData } from "..\\requests\\models\\ResponseData";  
```

But I expect this path /, it works correctly if I generate it in Linux: 
```Typescript
import { useQuery, useMutation, UseQueryOptions, UseMutationOptions } from "@tanstack/react-query";
import { ValidationError } from "../requests/models/ValidationError";
import { ResponseData } from "../requests/models/ResponseData";  
```
here is my syntax in package.json:
```json
"generate-client": "openapi-rq -i http://127.0.0.1:8082/openapi.json -o ./app/api --indent 2 --postfixServices \"\" ",
``` 

how do i fix it?

**Comment by 7nohe** [comment]
It was resolved in #23 and is included in v0.4.1.

**Comment by TheOdemMove** [comment]
Thanks!
